pkg/apis/serving/v1alpha1/servingruntime_types.go

```diff
@@ -126,6 +130,10 @@ type ServingRuntimeSpec struct {
 	// +optional
 	ProtocolVersions []constants.InferenceServiceProtocol `json:"protocolVersions,omitempty"`
 
+	// Set WorkerSpec to enable multi-node/multi-gpu
+	// +optional
+	WorkerSpec *WorkerSpec `json:"workerSpec,omitempty"`
```

Comment:
Is `WorkerSpec` required to be specified to turn on multi-node/multi-gpu ?

Reply:
Yes, specifying WorkSpec means it will use worker nodes (additional deployment for worker node).

Reply:
Since I see you already implemented a validator... I wonder, should we allow this going from `nil`->`non-nil` and vice-versa?

If people edit an existing SR and either set, or delete the field they are basically "converting" it. It could be a complication at ISVC side where the reconciliation happens.

And since this is also a new field in the ISVC, people may have it `nil` at the ServingRuntime, but may accidentally (or intentionally) set to non-nil in the InferenceService (or vice-versa). This could be understood as a non multi-node/multi-gpu ServingRuntime which is overriden ("promoted") as a multi-node/multi-gpu one in the ISVC side (or vice-versa). You may end-up in such situation (either accidentally or intentionally) if the SR is edited. I'm not sure if we should be allowing this complexity, but for first iteration, I'd suggest that we don't allow it and validate it.